mobius/src/motifs/halloween.rs

```diff
@@ -191,10 +198,14 @@ pub fn bone(length: f64) -> ClineArcTile {
     let circle_bottom_left = Circle::new(center_bottom_left, 1.0);
     let circle_bottom_right = Circle::new(center_bottom_right, 1.0);
 
-    let arc_top_left = CircularArc::new(circle_top_left, 0.0, FRAC_PI_2, 3.0 * FRAC_PI_2);
-    let arc_top_right = CircularArc::new(circle_top_right, -FRAC_PI_2, FRAC_PI_2, PI);
-    let arc_bottom_left = CircularArc::new(circle_bottom_left, FRAC_PI_2, PI, TAU);
-    let arc_bottom_right = CircularArc::new(circle_bottom_right, -PI, -FRAC_PI_2, FRAC_PI_2);
+    let angles_top_left = ArcAngles::new(0.0, FRAC_PI_2, 3.0 * FRAC_PI_2).unwrap();
```

Comment:
In complex motifs where geometry can't be shared, this is a lot of boilerplate. 
Maybe I should add a `CircularArc::from_params(Complex, f64, f64, f64, f64)` constructor for convenience? it would just delegate to the underlying `Circle` and `ArcAngles` constructors.

Reply:
Well... if so, then it would have to consolidate any errors from Circle and ArcAngles. That feels like it's getting beyond the scope of this PR.

That said, I should add more error handling for circles to detect degenerate circles (radius 0, bad float params, etc). But as a separate PR.